torch/_prims_common/wrappers.py

```diff
@@ -291,6 +291,21 @@ def _fn(*args, out=None, **kwargs):
     return _out_wrapper
 
 
+# Similar to out.defined() from core (only use for functions decorated with
+# out_wrapper).
+def out_defined():
+    stack = inspect.stack()
+    # Gets access to the wrapper function
+    _fn = stack[2]
+    # Checks that we're inspecting _fn from out_wrapper
+    assert _fn.function == "_fn"
+    # Checks that out arg is there since get() returns None on failure
+    argvalues = inspect.getargvalues(_fn.frame)
+    assert "out" in argvalues.args
+    # Checks whether out is not None
+    return argvalues.locals.get("out") is not None
```

Comment:
`fft` tests started failing after I added code copying the `conj` bit in `out_wrapper`. After investigating, I discovered that some `fft` ops have conditional logic for `conj` based on `out.defined()`. This adds a similar function on the ref side such that it's still possible to use `out_wrapper`.